### DISCUSSION TITLE
Do not ask for capacity on bicycle wheelbenders (bicycle_parking=wall_loops) ?

### DISCUSSION THREAD

**Discussion by mnalis** [streetcomplete/StreetComplete#3704] [Feedback]
I'd propose to disable quest for bicycle parking capacity if the bicycle parking is marked with `bicycle_parking=wall_loops`.

The reason is that i breaks the [quest guidelines](https://github.com/streetcomplete/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete), especially:
- 🤔 No unanswerable quests
- 🐿️ Easy answer: Users are out and about and impatient. A quick, straightforward and clear answer must be possible

For example (taken from https://sindikatbiciklista.hr/zagrebacke-spirale/ which has more example images)

![spirale-05-700x933](https://user-images.githubusercontent.com/156656/151822637-8c0837d5-ed8f-401f-b16a-6ba8f3e6acdd.jpg)

As shown in the picture, it is quite impossible to say what number of bikes can be parked. Naively, one can count number of loops (as theoretically, one wheel can fit per one loop) and get for example number like `30`. But as shown in picture, to lock that wheel, you need to use up _at least_ an additional one or two loops, which reduces number to `10`-`15`. But even that does not account for handlebars, which won't fit unless bicycles are ideally interleaved from both sides of loops, thus the realistic number is more likely about `4`-`6`. 
And of course, if one **actually cares for his bike**, only way to correctly park on wheelbender is to use your  bicycle stand and park _sideways_, in which case answer is `1`-`2`, depending. 
![spirale-13-468x351](https://user-images.githubusercontent.com/156656/151827070-b6c61877-1f5a-4571-a4c8-3b0d8276c59c.jpg)

Thus, as the question in **extremely subjective** with high range of uncorrelated answers, and **very hard and time consuming to answer**, I propose that `capacity` quest should be **_disabled for_** `bicycle_parking=wall_loops`. What do others think?

**Comment by matkoniecz** [comment]
In this case I think it is fairly easy to estimate actual capacity in the intended use, though it may be too much given that app should not assume any specialistic knowledge.

**Reply by mnalis** [reply]
Out of curiosity, what would be your "fairly easy" estimate for spiral wheelbender in the pictures above? One parking slot for each ring? Half the number of rings? One third of the number of rings (as in top picture)? Something else? 
E.g. if there was `20` such loops in a spiral, what number would _you_ put as the answer in the quest?

<details>
<summary>spoiler</summary>

For me personally (and majority of cycling people I know), I'd only ever park at wheelbender as depicted in lower picture, so my answer would be `1` - or `2` if other side of spiral was accessible too. But that is _with_ specialist bike-owner knowledge (i.e. knowing that I will likely be damaging my bike if I park otherwise at wheelbender). Someone who has never used a bike would probably not know that, so they'd I guess most likely answer `20`. 

Which is a **huge** difference...

</details>

**Reply by matkoniecz** [reply]
I would answer 1 per 5 rings (in use as wheelbender), and this is how most people use it.

Personally I would park as in image below, but marking it as `bicycle_parking=stands capacity=1` seems to be a poor solution.

**Reply by mnalis** [reply]
Yeah, well I certainly wouldn't _mistag_ it as [bicycle_parking=stands](https://wiki.openstreetmap.org/wiki/Key:bicycle_parking) (as they are not `stands`, i.e. it is not possible to lean a bike on it _nor_ to lock both a wheel and frame with single U-lock).

But yeah if it was accessible from one side only (i.e. other side of wheelbender was against the wall), and quest was presented to me, I would tag it as `bicycle_parking=wall_loops` + `capacity=1` - as that is the the way I think the most **intended users of that facility** (i.e. non-clueless bicycle owners who care about their bike) are most likely to use it (in my experience). 

The problem I see is that most SC users who do not own a bicycle would be **clueless** about it (just like the city management which approved such bicycle parkings to be installed - but I digress) and thus would probably enter completely wrong values like `20`, as _"one pushes the wheel between rings, and there are 20 rings, so you can park 20 bicycles there, right?"_  -- which is why I suggested skipping `capacity` quest in `wall_loops` case to avoid such problematic answers.

<details>

The issue is technically similar as when someone who do now know/care about length of the car would mark this https://github.com/streetcomplete/StreetComplete/issues/3686#issuecomment-1024936939 as a valid parking space (for those not knowing much of cars, there is not enough space to fully park the car there; so it would be half on the road and thus at high risk of being damaged if it tried to park there). The difference is in magnitude of error - it user miscounts such `fire_hydrant` space as valid parking space, it would in most cases be a **tiny error** (_"oh, ok, there is actually 19 car parking spaces instead of 20 - no big deal"_). But if the user mistags `wheel_bender`, the **error is huge** (_"ooops, there is actually only 1 bicycle parking space, not 20 of them"_)

</details>

**Comment by westnordost** [comment]
It's about for how many it is intended. It might be a bit difficult to estimate how many can exactly park at such a spiral, but I think an estimate is better than nothing.

**Reply by mnalis** [reply]
That was my main point, that the estimates **vary so widely** (someone might might mark it as `1` or `2`, while other user as `10` and yet another as `30`) that it is likely providing **more confusion then help** - that is more than order of magnitude "error". 

And (another big concern to me) is it is **stressful** for users , while providing **little if any** useful data (due to its high imprecision / ambiguity). As quest guidelines say: _"quick, straightforward and clear answer must be possible"_ - this fails on all three accounts: it is neither quick, nor straightforward, and answer is highly subjective and ambiguous instead of clear.

IMHO it is putting way too much pressure on users for too little gain in that specific case. Just my opinion, of course...

**Reply by lynxdingo** [reply]
On the wiki there is the key-value pair `bicycle_parking=wave`, though as the [wiki](https://wiki.openstreetmap.org/wiki/Talk:Key:bicycle_parking#W_rack_.2F_wave_.2F_ribbon_.2F_wave_.2F_serpentine_as_own_value.3F) points out one should use `bicycle_parking=wall_loops`. Thus in such a situation while removing ambiguity, the best solution I see is `capacity=*`. So if one can already enter the value `*`, the issue should be fixed. Or on the other hand, maybe add an dropdown/info option for such bicycle_parking situations, to explain to the unknown user of this possibility.

**Reply by mnalis** [reply]
@lynxdingo `*` in `capacity=*` is meant as a wildcard (i.e. meaning "any value"), and not as a literal character `*`. While there are 16 [uses worldwide](https://taginfo.openstreetmap.org/tags/capacity=*) of literal value `*` for `capacity` tag, that is probably due to misunderstanding and not to documented usage. Anyway, StreetComplete forces numerical entry in that quest, so even if one wanted, entering literal character `*` is forbidden.

**Reply by mnalis** [reply]
Anyway, for others with same problem; my solution to too much stress about that quest was to use [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) and edit the quest so bicycle count is **not** asked at all for `wall_loops`  (just like it wasn't asked on `floor` and `informal` already)

**Comment by mnalis** [comment]
- ref: https://github.com/streetcomplete/StreetComplete/issues/6206
